src/js/jsx/search/SearchBar.jsx

```diff
@@ -432,13 +449,21 @@ define(function (require, exports, module) {
                 category = "category-" + type;
 
             // Seeing if filters are active or not for analytics
-            if (filters.length === 0) {
+            if (filters && filters.length === 0) {
```

Comment:
Is `filters` guaranteed to be an array? If so, the first conjunct is unnecessary. If not, you probably want `!filter || filters.length === 0` instead?